packages/sdk-drop-in/tests/iframe.test.ts

```diff
@@ -0,0 +1,116 @@
+  }
+
+  return {
+    entity_user_id: entityUserId,
+    client_id: clientId,
+    client_secret: clientSecret,
+    grant_type: 'entity_user',
+  };
+};
+
+// console.log('ToDo: check where we store the token', getFileConfig());
+// console.log('ToDo: check where we store the token', geEnvConfig());
+
+console.log(process.env);
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+console.log(import.meta.env || "can't find env");
+
+const consumerPage = '/monite-iframe-app-consumer';
+
+const routingPaths: Record<WidgetType, string> = {
+  payables: '/payables',
+  receivables: '/receivables',
+  counterparts: '/counterparts',
+  products: '/products',
+  tags: '/tags',
+  'approval-policies': '/approval-policies',
+  onboarding: '/onboarding',
+};
+
+test.describe('Monite Iframe Integration', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`${consumerPage}${routingPaths.receivables}`);
+  });
+
+  test('should see the sales and navigate inside the iframe', async ({
+    page,
+  }) => {
+    const iframe = page.frameLocator('iframe');
+
+    await iframe.getByRole('heading', { name: 'Sales' }).click();
+    await iframe.getByRole('tab', { name: 'Invoices' }).click();
+    await iframe.getByRole('tab', { name: 'Quotes' }).click();
+    await iframe.getByRole('tab', { name: 'Credit notes' }).click();
+    await iframe.getByRole('button', { name: 'Create Invoice' }).click();
+    await iframe.getByLabel('close').click();
+  });
+
+  test('test the theme switcher and tabs', async ({ page }) => {
+    await page.getByRole('button', { name: 'Material UI' }).click();
+    await page.getByText('Theme').click();
+    await page.getByRole('button', { name: 'Material UI' }).click();
+    await page.getByRole('menuitem', { name: 'Monite' }).click();
+    await page.getByRole('button', { name: 'Monite' }).click();
+    await page.getByLabel('Dark Mode').check();
+    await page.locator('.MuiBackdrop-root').click();
+    await page.getByRole('button', { name: 'Settings' }).click();
+    await page.getByRole('button', { name: 'Roles' }).click();
+    await page.getByRole('button', { name: 'Tags' }).click();
+
+    const iframe = page.frameLocator('iframe');
+    await iframe.getByRole('button', { name: 'Create new tag' }).click();
+    await iframe.getByRole('button', { name: 'Cancel' }).click();
+  });
+
+  type WidgetTest = {
+    path: string;
+    name: string;
+  };
+
+  const widgetTests: WidgetTest[] = [
+    { path: routingPaths.payables, name: 'Payables' },
+    { path: routingPaths.counterparts, name: 'Counterparts' },
+    { path: routingPaths.products, name: 'Products' },
+  ];
+
+  widgetTests.forEach(({ path, name }) => {
+    test(`should see the ${name.toLowerCase()} tab`, async ({ page }) => {
+      await page.goto(`${consumerPage}${path}`);
+      const iframe = page.frameLocator('iframe');
+      await iframe.getByRole('heading', { name }).click();
```

Comment:
We don't particularly have to care about roaming in the Consumer App. But we should care that the buttons in the menu lead to the right Super Components inside iframe:

```tsx
await page.getByRole('button', { name }).click(); // click on the menu menu item
await iframe.getByRole('heading', { name }).toBeVisible(); // check if heading is rendered
```